staging/src/kubevirt.io/client-go/Makefile

```diff
@@ -0,0 +1,5 @@
+include $(addprefix ../../../../vendor/github.com/openshift/build-machinery-go/make/, \
```

Comment:
why do we need a makefile to accomplish this?

Reply:
Removed